iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractLoadingLimits.java

```diff
@@ -59,9 +61,14 @@ public int getAcceptableDuration() {
         public boolean isFictitious() {
             return fictitious;
         }
+
+        @Override
+        public void setValue(double temporaryLimitValue) {
+            this.value = temporaryLimitValue;
+        }
     }
 
-    AbstractLoadingLimits(OperationalLimitsGroupImpl owner, double permanentLimit, TreeMap<Integer, TemporaryLimit> temporaryLimits) {
+    AbstractLoadingLimits(Validable validable, OperationalLimitsGroupImpl owner, double permanentLimit, TreeMap<Integer, TemporaryLimit> temporaryLimits) {
```

Comment:
The `Validable` is useful to have more detailed messages.
But it can be retrieved from the `OperationalLimitsGroupImpl` with `group.getValidable()`.

@nao1345678: can you remove the `AbstractLoadingLimits.validable` attribute (and this parameter)?